rust/flatbuffers/src/vector.rs

```diff
@@ -52,7 +52,7 @@ impl<'a, T: 'a> Vector<'a, T> {
     }
 }
 
-impl<'a, T: 'a> Vector<'a, T> {
+impl<'a, T: 'a> Vector<'a, T: Follow<'a> + 'a> {
```

Comment:
Should be ` impl<'a, T: 'a + Follow<'a>> Vector<'a, T> {`

Reply:
Woops What I got for editing on the road :)